client/i18n/locales/english/translations.json

```
@@ -113,6 +113,7 @@
     "claim-legacy": "Once you've earned the following freeCodeCamp certifications, you'll be able to claim the {{cert}}:",
     "for": "Account Settings for {{username}}",
     "sound-mode": "This adds the pleasant sound of acoustic guitar throughout the website. You'll get musical feedback as you type in the editor, complete challenges, claim certifications, and more.",
+    "sound-volume": "Volume:",
```

```suggestion
    "sound-volume": "Campfire Volume:",
```

Just to make it perfectly clear that the volume control is for the campfire sounds.